### DISCUSSION TITLE
Creating a Nested Translation

### DISCUSSION THREAD

**Discussion by ignatius-n** [spatie/laravel-translation-loader#167] [Bugs]
Hello,
I'm wondering how can I create a nested translation such as
`"admin.cruds.users.fields.firstName"`

trying to create it as

```
group => 'admin.cruds.users.fields' 
key => 'firstName'
text => 'John'
```
// OR
```
group => 'admin.cruds.users.fields.'
key => 'firstName'
text => 'John'
```
doesn't return a translation but something like:

```
group => 'admin'
key => 'firstName'
text => 'John'
```
works just fine

**Comment by marc-soriano** [comment]
Same problem here. Is there any way to work with subfolders?

**Comment by kevinartimedia** [comment]
help us! :(

**Comment by mgcodeur** [comment]
@ignatius-n @marc-soriano @kevinartimedia The group cannot be nested, but the key can be.

Example:

group: backend  
key: menu.home  

=> __('backend.menu.home')
